project/accounts/templates/accounts/users/password_reset_done.html

```diff
@@ -1,6 +1,6 @@
 {% extends 'accounts/register/entry_base.html' %}
 
-{% block content %}
+{% block entry_content %}
```

Comment:
This should remain as `content`
```suggestion
{% block content %}
```